reactDashboard/src/components/DataCharts/DataChartsUtils.js

```diff
@@ -17,18 +17,18 @@ const keyDates = [
   { date: Date.parse("2020-10-09"), name: "BARS CLOSE" },
 ];
 
-function getDateLine({ date, name }, index) {
+function getDateLine( { date, name }, index, darkmode) {
   return {
     type: "line",
     drawTime: "afterDatasetsDraw",
     mode: "vertical",
     scaleID: "x-axis-0",
-    borderColor: "rgba(0,0,0,0.25)",
+    borderColor: darkmode ? "#f2f2f2" : "rgba(0,0,0,0.25)",
     borderWidth: 2,
     value: date,
     label: {
-      backgroundColor: "#007EB9",
-      fontColor: "#ffffff",
+      backgroundColor: darkmode? "#c1def1" : "#007EB9",
+      fontColor: darkmode? "#121212" : "#ffffff",
```

Comment:
Very nitpicky but shouldn't there be a space between "darkmode" and "?" for lines 30 and 31?

Reply:
There's a couple of formatting issues in this file - perhaps run prettier on it ?

Reply:
@rcarmitage That's not being nitpicky- thanks for catching that.
@donal-s ✅